src/dex/solidly/config.ts

```diff
@@ -159,6 +159,18 @@ export const SolidlyConfig: DexConfigMap<DexParams> = {
       feeCode: 0,
     },
   },
+  PharaohV1: {
```

Comment:
@JackieJoo I would suggest to rename Pharaoh (which is UniV3 fork) to PharaohV2 to have consistency with Ramses.